lib/node_modules/@stdlib/repl/lib/keybindings.js

```diff
@@ -0,0 +1,347 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Table mapping of REPL actions and their corresponding keybinding triggers.
+*
+* @name KEYBINDINGS
+* @type {Object}
+*/
+var KEYBINDINGS = {
+	'moveRight': [
+		{
+			'name': 'right',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'f',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'moveLeft': [
+		{
+			'name': 'left',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'b',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'moveWordRight': [
+		{
+			'name': 'right',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'f',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'moveWordLeft': [
+		{
+			'name': 'left',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'b',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
+	],
+	'moveBeginning': [
+		{
+			'name': 'home',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'a',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'moveEnd': [
+		{
+			'name': 'end',
+			'ctrl': false,
+			'shift': false,
+			'meta': false
+		},
+		{
+			'name': 'e',
+			'ctrl': true,
+			'shift': false,
+			'meta': false
+		}
+	],
+	'tab': [
+		{
+			'name': 't',
+			'ctrl': false,
+			'shift': false,
+			'meta': true
+		}
```

Comment:
What should be the expected behavior with this key? When trying it out, seemed to overlap with TAB completion behavior.